main.tf

```diff
@@ -348,11 +347,13 @@ module "extensions" {
   # bastion
   create_bastion_host = var.create_bastion_host
   bastion_public_ip   = local.bastion_public_ip
+  bastion_user        = var.bastion_user
```

Comment:
How are you ensuring that the bastion can have a different user? The use case is to use the default 'ubuntu' as a user for the ubuntu image. We need to make a change in the bastion module so that an Ubuntu image can also be used.

Reply:
It doesn't go that far yet - just connecting config in the module. It could still support a non-default user on a custom OL image this way, or work with a provided bastion host. Future enhancements to bastion will be ready to use with the variables in place.

Reply:
Added a note on setting to "opc" for supported images.